Tests/ArgumentParserEndToEndTests/TransformEndToEndTests.swift

```diff
@@ -0,0 +1,157 @@
+//===----------------------------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Argument Parser open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import ArgumentParserTestHelpers
+import ArgumentParser
+
+final class TransformEndToEndTests: XCTestCase {
+}
+
+fileprivate enum FooBarError: Error {
+    case outOfBounds
+}
+
+fileprivate protocol Convert {
+    static func convert(_ str: String) throws -> Int
+}
+
+extension Convert {
+    
+    
+    static func convert(_ str: String) throws -> Int {
```

Comment:
Nit: Close this up, and convert to two space indents in this file.
```suggestion
    static func convert(_ str: String) throws -> Int {
```